arangod/Pregel/Conductor.cpp

```diff
@@ -402,7 +402,7 @@ void Conductor::startRecovery() {
   _statistics.reset();
 
   TRI_ASSERT(SchedulerFeature::SCHEDULER != nullptr);
-  boost::asio::io_service* ioService = SchedulerFeature::SCHEDULER->ioService();
+  boost::asio::io_service* ioService = SchedulerFeature::SCHEDULER->ioContext();
```

Comment:
Should this not be `asio_ns::` now?
And io_context?

Reply:
Yes, it should be asio_ns, fixed.
io_context is aliased.